quamash/_windows.py

```diff
@@ -15,22 +15,23 @@
 
 import math
 
-from . import QtCore
+from . import _make_signaller
 from ._common import with_logger
 
 UINT32_MAX = 0xffffffff
 
 
-class _ProactorEventLoop(QtCore.QObject, asyncio.ProactorEventLoop):
+class _ProactorEventLoop(asyncio.ProactorEventLoop):
 
 	"""Proactor based event loop."""
 
 	def __init__(self):
-		QtCore.QObject.__init__(self)
-		asyncio.ProactorEventLoop.__init__(self, _IocpProactor())
+		super().__init__(_IocpProactor())
 
-		self.__event_poller = _EventPoller()
-		self.__event_poller.sig_events.connect(self._process_events)
+		self.__event_signaller = _make_signaller(self._qtcore, list)
```

Comment:
I personally would go for passing it through super() and let the base class deal with setting the ``_qtcore`` attribute. Make it part of the ("protected", in C++ terms) interface a base class for QEventLoop must implement.

(Or just document it.)